src/main/java/mate/academy/Main.java

```diff
@@ -1,20 +1,40 @@
+/**
+ * Contains the main class of the application.
+ */
 package mate.academy;
 
-import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
 import mate.academy.service.AuthenticationService;
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
 
-public class Main {
-    public static void main(String[] args) {
-        AuthenticationService authenticationService = new AuthenticationServiceImpl();
+import java.util.logging.Logger;
```

Comment:
using java logger is not very good solution. 
try to use `logback` or `log4j2` instead
```suggestion
import java.util.logging.Logger;
```